homeassistant/components/plaato/sensor.py

```diff
@@ -0,0 +1,142 @@
+"""Support for Plaato Airlock sensors."""
+
+import logging
+
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from . import (
+    ATTR_ABV, ATTR_BATCH_VOLUME, ATTR_BPM, ATTR_CO2_VOLUME, ATTR_TEMP,
+    ATTR_TEMP_UNIT, ATTR_VOLUME_UNIT, DOMAIN as PLAATO_DOMAIN,
+    PLAATO_DEVICE_ATTRS, PLAATO_DEVICE_SENSORS, SENSOR as SENSOR_DOMAIN,
+    SENSOR_UPDATE)
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_KEY = '{}.{}'.format(PLAATO_DOMAIN, SENSOR_DOMAIN)
+
+
+async def async_setup_platform(hass, config, async_add_entities,
+                               discovery_info=None):
+    """Set up the Plaato sensor."""
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up Plaato from a config entry."""
+    devices = {}
+
+    def get_device(device_id):
+        """Get a device."""
+        return hass.data[PLAATO_DOMAIN].get(device_id, False)
+
+    def get_device_sensors(device_id):
+        """Get device sensors."""
+        return hass.data[PLAATO_DOMAIN].get(device_id)\
+            .get(PLAATO_DEVICE_SENSORS)
+
+    async def _update_sensor(device_id):
+        """Update/Create the sensors."""
+        if device_id not in devices and get_device(device_id):
+            entities = []
+            sensors = get_device_sensors(device_id)
+
+            for sensor_type in sensors.items():
+                entities.append(PlaatoSensor(device_id, sensor_type))
+
+            devices[device_id] = entities
+
+            async_add_entities(entities, True)
+        else:
+            for entity in devices[device_id]:
+                entity.async_schedule_update_ha_state()
+
+    hass.data[DATA_KEY] = async_dispatcher_connect(
+        hass, SENSOR_UPDATE, _update_sensor
+    )
+
+    return True
+
+
+class PlaatoSensor(Entity):
```

Comment:
Please add property `should_poll` and have it return `False`

Reply:
Done